app/helpers/application_helper/toolbar/miq_template_center.rb

```diff
@@ -71,7 +71,8 @@ class ApplicationHelper::Toolbar::MiqTemplateCenter < ApplicationHelper::Toolbar
           'fa fa-search fa-lg',
           N_('Check Compliance of the last known configuration for this Template'),
           N_('Check Compliance of Last Known Configuration'),
-          :confirm => N_("Initiate Check Compliance of the last known configuration for this Template?")),
+          :confirm => N_("Initiate Check Compliance of the last known configuration for this Template?"),
+          :klass   => ApplicationHelper::Button::MiqCheckCompliance),
```

Comment:
@romanblanco looks like this toolbar is not in use, that toolbar was in use back when going thru relationships like Hosts to its list of templates and then clicking on a template we used to load non-explorer screen. You should delete that toolbar.